themes/tiddlywiki/vanilla/base.tid

```diff
@@ -92,7 +91,7 @@ pre {
 	white-space: pre-wrap;
 	background-color: <<colour pre-background>>;
 	border: 1px solid <<colour pre-border>>;
-	padding: 0 3px 2px;
+	padding: 0.5em;
```

Comment:
ouch. .. ok :) ... 

To be honest, I personally like the existing padding more than 0.5 em. So for me the plugin would need to be changed. ...

But there should be no need to change the base CSS. .. The plugin just needs to make sure, that every code block uses a `.hljs` class as the highest priorty. So imo there is an issue with the plugin!